cli/src/funTest/kotlin/ExamplesFunTest.kt

```diff
@@ -162,6 +170,35 @@ class ExamplesFunTest : StringSpec() {
             report shouldHaveSize 1
         }
 
+        "emails can be send using notifications.kts" {
+            val greenMail = GreenMail()
+            greenMail.setUser("no-reply@ossreviewtoolkit.org", "no-reply@ossreviewtoolkit.org", "pwd")
+            greenMail.start()
+
+            val ortResult = File("src/funTest/assets/semver4j-analyzer-result.yml").readValue<OrtResult>()
+            val notifier = Notifier(
+                ortResult, NotifierConfiguration(
+                    EmailConfiguration(
+                        hostName = "localhost",
+                        port = 3025,
+                        username = "no-reply@ossreviewtoolkit.org",
+                        password = "pwd",
+                        useSsl = false,
+                        fromAddress = "no-reply@ossreviewtoolkit.org"
+                    )
+                )
+            )
+
+            notifier.run(takeExampleFile("notifications.kts").readText())
```

Comment:
Extract `takeExampleFile("notifications.kts").readText()` into separate variable would increase readability and align with other test for rules